openhcl/underhill_core/src/worker.rs

```diff
@@ -1393,6 +1393,19 @@ async fn new_underhill_vm(
         }
     }
 
+    // Always enable default boot always attempt for non-Trusted Launch VMs.
```

Comment:
>// Always enable default [](http://example.com/codeflow?start=4&length=24)

want to comment this as temporary until we can reliably depend on dps.general.default_boot_always_attempt coming from the host?  (internally, this means AH2025 host saturation with a fix)